sdk/attestation/azure-security-attestation/README.md

```diff
@@ -116,8 +116,7 @@ authentication, the documentation for that API will reflect that the attestation
 
 To interact with the authenticated APIs supported by the Azure Attestation service, your client must present an Azure Active Directory bearer token to the service.
 
-The simplest way of providing a bearer token is to use the  `ClientSecretCredential` authentication method by providing client secret credentials is being used in this
-getting started section, but you can find more ways to authenticate with [azure-identity][azure_identity].
+The simplest way of providing a bearer token is to use the `DefaultAzureCredential` authentication method is being used in this getting started section, but you can find more ways to authenticate with [azure-identity][azure_identity].
```

Comment:
This sentence is awkward to read, probably because of a typo.

> use the `DefaultAzureCredential` authentication method is being used in this getting started section

Did you mean this?

```suggestion
The simplest way of providing a bearer token is to use the `DefaultAzureCredential` authentication method which is being used in this getting started section, but you can find more ways to authenticate with [azure-identity][azure_identity].
```